src/api/index.js

```diff
@@ -81,8 +81,8 @@ export const saveManagementConsiderations = (
     managementConsiderations.map(async consideration => {
       if (uuid.isUUID(consideration.id)) {
         const { id, ...values } = consideration
-        const { data } = await axios.post(
-          API.CREATE_RUP_MANAGEMENT_CONSIDERATION(planId),
+        const { data } = await axios.put(
+          API.UPDATE_RUP_MANAGEMENT_CONSIDERATION(planId, consideration.id),
           values,
           getAuthHeaderConfig()
         )

```

Comment:
Shouldn't have had to flip this around as UUIDs are used exclusively on the frontend. Let me know what error you were getting; it may have been from something else.

Reply:
That logic doesn't make sense the way it was setup initially. If a UUID is attached to the record then it means it's already been persisted to the DB so any further changes should be updates (`PUT`).

Reply:
UUID means it is **not** persisted. If you look at the click handler for the "Add" button you'll see it creates new management considerations with a UUID.

Reply:
My apologies I did not read your first message before adding that comment. I don't recall easier that behaviour though so I will look at this again.

Reply:
@mccullochk My mistake, it looks like the UUID for new management considerations never got implemented. Take a look at https://github.com/bcgov/range-web/blob/dev/src/components/rangeUsePlanPage/additionalRequirements/index.js#L54 for an example if you want to add it. Should be added to https://github.com/bcgov/range-web/blob/dev/src/components/rangeUsePlanPage/managementConsiderations/index.js#L71.